crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2503,6 +2745,13 @@ impl AuthorityPerEpochStore {
                         .entry(deferral_key)
                         .or_default()
                         .push(tx.clone());
+                    if let Some(key) =
+                        tx.0.transaction
+                            .executable_transaction_digest()
+                            .map(TransactionKey::Digest)
```

Comment:
I count 4 or 5 instances of this pattern - might be worth adding a From impl so you can just call into()